ubuntu/golang/binaris-1.9.2.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+set -eu
+
+# Install Go under /usr/local.
+curl -sL https://redirector.gvt1.com/edgedl/go/go1.9.2.linux-amd64.tar.gz | tar -C /usr/local/ -xzf -
```

Comment:
Reading the docs I see that it says to add the GO binary to your PATH after installation. Is this not needed for us?

> Add /usr/local/go/bin to the PATH environment variable. You can do this by adding this line to your /etc/profile (for a system-wide installation) or $HOME/.profile: